### PR TITLE
add dependencies to CommonTools/Utils BuildFile as needed in src/interface

### PR DESCRIPTION
**Pull request by davidlange6**
FWCore/Parameter  noted as missing in modules build..I've added all interface/src includes

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: CommonTools/Utils/BuildFile.xml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<use name="DataFormats/Common"/>
+<use name="DataFormats/Math"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/Reflection"/>
+<use name="FWCore/SOA"/>
+<use name="FWCore/Version"/>
```
Comment (slava77) [inline CommonTools/Utils/BuildFile.xml:6]: this is apparently fake: it's included only in CommonTools/Utils/src/ExpressionEvaluator.cc, but the `getReleaseVersion` method is not used.

Comment (davidlange6) [reply CommonTools/Utils/BuildFile.xml:6]: Seems to be the case..

**Comment by davidlange6** [comment]
please test

**Comment by santocch** [comment]
+1

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/31291	a5c777c 
- technical; in line with the title/description and now the code review

**Comment by qliphy** [comment]
+1

**Pull request merged**